### PR TITLE
feat: modal migration UI 

### LINKED ISSUE #3117: [UI] Migration Notification Screen
# Specs
https://www.notion.so/jan-ai/UI-Migration-notification-screen-b11fabedd05b40839afa07275022c440?pvs=4

# Design
https://www.figma.com/design/DYfpMhf8qiSReKvYooBgDV/Jan-App-(3rd-version)?node-id=5360-105063&t=xJCIK6XI2nLkG2VX-4

### PR DESCRIPTION
**Pull request by urmauur**
## Describe Your Changes

image
image

## Fixes Issues

- Closes #3117 
- Closes #

## Self Checklist

- [ ] Added relevant comments, esp in complex areas
- [ ] Updated docs (for bug fixes / features)
- [ ] Created issues for follow-up changes or refactoring needed

### CODE REVIEW

File: web/hooks/useMigratingData.ts [hunk 1]
```diff
@@ -33,64 +50,70 @@ const useMigratingData = () => {
...
-        const modelId: string | undefined = thread.assistants[0]?.model?.id
-        if (!modelId || modelId.trim().length === 0 || modelId === '*') {
-          console.error(
-            `Ignore thread ${thread.id} because modelId is not found`
-          )
-          continue
-        }
-        const threadTitle: string = thread.title ?? 'New Thread'
-        const instruction: string = thread.assistants[0]?.instruction ?? ''
+      if (!threadsMessagesMigrationSuccess) {
```
Comment (namchuai) [inline web/hooks/useMigratingData.ts:53]: I think we can use early return here.
if (threadsMessagesMigrationSuccess) return

**Comment by namchuai** [comment]
looks good to me but I think we need @Van-QA to test it

**Comment by Van-QA** [comment]
@urmauur. The blue check from your screenshot should green to indicate the completion right? can you double check the figma (it's green)
image

**Comment by urmauur** [comment]
> @urmauur. The blue check from your screenshot should be green to indicate the completion right? can you double check the Figma? There should be white background in the retry as well image

@Van-QA updated design, and since checkbox is using variable css `primary-bg` and user can custom it, doesn't make sense to create checkbox with green bg, i already talk with Ashley

Screenshot 2024-07-12 at 00 26 02

**Pull request merged**